Re-paths gloves

PsiOmegaDelta opened the pull request:
Yellow gloves are no longer inherently insulated.
Black gloves are no longer inherently fire-resistant.

These are now proper, separate subtypes from specific coloring itself.

PsiOmegaDelta committed:
Re-arranges glove paths.

Yellow gloves are no longer inherently insulated.
Black gloves are no longer inherently fire-resistant.

PsiOmegaDelta committed:
Washing machines can again color gloves, because they're no longer magical.

PsiOmegaDelta committed:
Removes some now unused glove icons. Adds a default missing-icon state.

PsiOmegaDelta committed:
Vox gloves caused a mess, due to being an extending /gloves/yellow type.

Mass-replaces all yellow glove instances.

mwerezak commented on code/modules/clothing/spacesuits/alien.dm:
```diff
@@ -106,7 +106,7 @@
 	item_state = "vox-casual-2"
 
-/obj/item/clothing/gloves/yellow/vox
+/obj/item/clothing/gloves/color/insulated/vox
```
No real reason to make the inherit from insulated, and I don't think the sprite is all that amenable to recolourization.

Kasuobes commented:
This will make insulated gloves available from the loadout. Intended?

Techhead0 commented:
From reading the PR, it looks like only the basic colored, latex, rainbow, and evening gloves are available. Which also means that the thick gloves are *no longer* available.

I also don't see why insulated gloves are a subtype of `gloves/color`, or why evening gloves aren't.

mwerezak committed:
Makes some glove paths more logical

/obj/item/clothing/gloves/color/thick -> /obj/item/clothing/gloves/thick
/obj/item/clothing/gloves/swat -> /obj/item/clothing/gloves/thick/swat
/obj/item/clothing/gloves/combat -> /obj/item/clothing/gloves/thick/combat
/obj/item/clothing/gloves/botanic_leather -> /obj/item/clothing/gloves/thick/botany
/obj/item/clothing/gloves/color/insulated -> /obj/item/clothing/gloves/insulated

mwerezak committed:
Adds desaturated evening glove sprites

Makes them a subtype of colored gloves

mwerezak commented:
@PsiOmegaDelta https://github.com/PsiOmegaDelta/Baystation12/pull/39

mwerezak committed:
Makes budget gloves a little less terrible

PsiOmegaDelta commented:
Yes, I do want several notifications for the same thing. Thanks.

PsiOmegaDelta committed:
Expands the colorable glove macro.

PsiOmegaDelta commented:
It's only been awaiting a preview for 3 almost weeks now.

Techhead0 commented:
What's with the 200 line changes to Exodus-2 and the few thousand changes to the Kung-Peng maps?

PsiOmegaDelta committed:
Adjusts maps.

PsiOmegaDelta commented:
Maps "corrected" (or rather, someone appears to have committed maps full of newline changes which my PR simply made evident).
